robotbase/src/main/java/frc/robot/commands/pivot/ShooterPivotIntakeFromSource.java

```diff
@@ -0,0 +1,26 @@
+package frc.robot.commands.pivot;
```

Comment:
This is fine for now. Eventually we'll have a command that will just set the rotations that are passed into the command (I haven't done that yet).